src/eckit/exception/Exceptions.cc

```diff
@@ -83,19 +83,7 @@ Exception::Exception(const std::string& w, const CodeLocation& location, bool qu
         Log::error() << "Exception: " << w << " " << location_ << std::endl;
     }
 
-#if 0
-    if (next_) {
-        Log::error() << "Exception: stack containts " << next_->what() << std::endl;
-    }
-    else
-    {
-        Log::error() << "Exception: stack is empty" << std::endl;
-    }
-#endif
-
     first() = this;
-
-    Log::status() << "** " << w << location_ << std::endl;
```

Comment:
This doesn't look to be the right change.

This just removes Log::status() from eckit exception. That is definitely wrong.

Please wrap Log::status() in the quiet check, not remove it.